content/en/docs/components/model-registry/installation.md

```diff
@@ -32,19 +32,7 @@ To deploy Model Registry UI, you just need to follow the [UI section](https://gi
 
 #### Installing on a Kubeflow Profile
 
-Kubeflow Central Dashboard uses [Profiles](/docs/components/central-dash/profiles/) to handle user namespaces and permissions. By default, the manifests deploy the Model Registry in the `kubeflow` namespace, to install a compatible version of Model Registry for Kubeflow, you should first head into the [istio overlay](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream/options/istio) and run the following commands:
-
-```shell
-kustomize set namespace <your-profile>
-kubectl apply -k .
-```
-
-Then head into the [db overlay](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream/overlays/db) and run the following commands:
-
-```shell
-kustomize set namespace <your-profile>
-kubectl apply -k .
-```
+Kubeflow Central Dashboard uses [Profiles](/docs/components/central-dash/profiles/) to handle user namespaces and permissions. To install a new Model Registry instance on the targeted profile, just follow the instructions under ["Install MR on a Kubeflow Profile"](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#installing-model-registry-on-a-kubeflow-profile) section.
```

Comment:
I would also clean up above "installing on Kubeflow platform" section.  I see 3 ways to install so far
* Install on Kubeflow Platform
* Install on Kubeflow profile
* standalone
first two instructions are in manifests, third one here. If there is a way reduce steps to two  like 
* "platform" vs "profile" a single recommended way to install MR
* standalone
would be ideal

Reply:
I would leave that for the team, we can do a follow up clean up, this PR is targeted to enhance MR UI installation, we can discuss how to further improve the rest of the docs in other task.